src/widgets/modalChildren/AutocompleteSelector.js

```diff
@@ -104,7 +102,7 @@ export const AutocompleteSelector = ({
         autoCorrect={false}
         autoFocus
         color={WHITE}
-        onChangeText={onChangeText}
+        onChangeText={inputValue => setQueryText(inputValue)}
```

Comment:
I prefer this to be defined as it was before. This is because whenever a re-render occurs, anonymous functions like this are re-created. It's a tiny overhead, but it's good to ensure we are always defining functions explicitly.

So as above,

```
const onChangeText = inputValue => setQueryText(inputValue)
...
onChangeText={onChangeText}
```

To be even better, we can use a react hook, and memoize the function, to cause even less overhead!

```
const onChangeText = React.useCallback(inputValue => setQueryText(inputValue), [])
...
onChangeText={onChangeText}
```

What this does is - React will store an object that will hold this function. Then, whenever re-renders occur, react will just give us back the same function, rather than creating a new one.

Reply:
Nice suggestion and great to remember the use of memoization. 
Done here 566f930